### DISCUSSION TITLE
All matched classess are updated

### DISCUSSION THREAD

**Discussion by jamir-hossain** [GrapesJS/grapesjs#4104] [Q&A]
If I update a class using selector manager from the side panel all the similar classes are also being updated. Is there any way to disable that? I just want to update the selected element's class.

Please check the screen record.
https://www.loom.com/share/ec3bbfc953ca4eff9c6271db8ddf4e94

Thanks in advance!

**Comment by artf** [comment]
Yeah, the default Selector Manager UI doesn't provide that, you should remove that selector and add a new one. You can also implement [your own Selector Manager UI](https://grapesjs.com/docs/modules/Selectors.html#customization)
